activiti-api-process-model/src/main/java/org/activiti/api/process/model/builders/TimerPayloadBuilder.java

```diff
@@ -0,0 +1,105 @@
+package org.activiti.api.process.model.builders;
+
+import java.util.Date;
+
+import org.activiti.api.process.model.payloads.TimerPayload;
+
+public class TimerPayloadBuilder {
```

Comment:
I think you can remove this class and use directly the payload.

Reply:
Removed here: 5475a36